package.json

```diff
@@ -8,20 +8,13 @@
     "test": "cross-env NODE_ENV=test ./node_modules/.bin/jest",
     "dist": "yarn compile && electron-builder",
     "dist:dir": "yarn dist --dir -c.compression=store -c.mac.identity=null",
-    "db-down":
-      "node node_modules/db-migrate/bin/db-migrate down --config config/database.json",
-    "db-up":
-      "node node_modules/db-migrate/bin/db-migrate up --config config/database.json",
-    "db-reset":
-      "node node_modules/db-migrate/bin/db-migrate reset --config config/database.json",
-    "db-create-config":
-      "cross-env NODE_ENV=dev babel-node scripts/create-db-config/create.js",
-    "db-migrate-create":
-      "node node_modules/db-migrate/bin/db-migrate create -e dev --config config/database.json",
-    "add-some-notes":
-      "cross-env NODE_ENV=$ENV babel-node scripts/convert-notes/convert.js scripts/convert-notes/our_small_dataset.json",
-    "add-many-notes":
-      "cross-env NODE_ENV=$ENV babel-node scripts/convert-notes/convert.js scripts/convert-notes/our_large_dataset.json",
+    "db-down": "node node_modules/db-migrate/bin/db-migrate down --config config/database.json",
+    "db-up": "node node_modules/db-migrate/bin/db-migrate up --config config/database.json",
+    "db-reset": "node node_modules/db-migrate/bin/db-migrate reset --config config/database.json",
+    "db-create-config": "cross-env NODE_ENV=dev babel-node scripts/create-db-config/create.js",
+    "db-migrate-create": "node node_modules/db-migrate/bin/db-migrate create -e dev --config config/database.json",
+    "add-some-notes": "cross-env NODE_ENV=$ENV babel-node scripts/convert-notes/convert.js scripts/convert-notes/our_small_dataset.json",
+    "add-many-notes": "cross-env NODE_ENV=$ENV babel-node scripts/convert-notes/convert.js scripts/convert-notes/our_large_dataset.json",
```

Comment:
not a big issue, but I've noticed our different linters seems to keep changing the formatting of these longer scripts, so we should probably try to figure out how to make it consistent.

Reply:
I believe it's a fight between prettier and npm. Prettier will try and format the package.json and then npm (whenever we install something) will remake the file in an unformatted fashion. 

We could probably get visual studio and prettier to ignore formatting this file. 

Or we could make the scripts smaller?

Reply:
Let's not try to make the scripts smaller... we've wasted enough time on that already. I've already added this file to files my prettier ignores so it stops trying to format it.

Reply:
?? Wait where's your prettier ignores? Can you add that to the project?

Reply:
It was in the settings of atom that I changed it. not sure how to do that for everyone.